fission-web-server/library/Fission/Web/Server.hs

```diff
@@ -179,10 +181,20 @@ bizServer ::
   , App.Domain.Retriever      t
   )
   => ServerT Fission.API m
-bizServer = IPFS.handler
-       :<|> App.handler
-       :<|> Heroku.handler
-       :<|> User.handler
-       :<|> Ping.handler
-       :<|> DNS.handler
-       :<|> Auth.UCAN.Verify.handler
+bizServer = v2 :<|> v_
+  where
+    v2 =   IPFS.handler
+      :<|> App.handler
+      :<|> Heroku.handler
+      :<|> User.handler
+      :<|> Ping.handler
+      :<|> DNS.handler
+      :<|> Auth.UCAN.Verify.handler
+
+    v_ =   IPFS.handler
+      :<|> App.handler
+      :<|> Heroku.handler
+      :<|> User.handler
+      :<|> Ping.handler
+      :<|> DNS.handler
+      :<|> Auth.UCAN.Verify.handler
```

Comment:
Duplicated because we expect the V2 version to change over time, but the unversioned one not to!